m4/enable_cpuset.m4

```diff
@@ -50,8 +50,11 @@ AC_DEFUN([PBS_AC_ENABLE_CPUSET],
     AS_IF([test ! -r /usr/include/cpuset.h],
       AC_MSG_ERROR([Missing cpuset header file]))
     AS_IF([test -r /usr/include/cpumemset.h],
-      [cpuset_flags="-DCPUSET_VERSION=2"],
-      [cpuset_flags="-DCPUSET_VERSION=4"])
+      [cpuset_flags="-DCPUSET_VERSION=2"; cpuset_v4=no],
+      [cpuset_flags="-DCPUSET_VERSION=4"; cpuset_v4=yes])
+      AS_IF([test x$cpuset_v4 = xyes],
+        AS_IF([test ! -r /usr/include/bitmask.h],
+	  AC_MSG_ERROR([Missing libbitmask or bitmask.h header file])))
```

Comment:
Please do not use tabs for indentation in the m4 files.

Reply:
Sorry for that - I've just pushed an update which eliminates any tabs. Set my editor to show tabs now.